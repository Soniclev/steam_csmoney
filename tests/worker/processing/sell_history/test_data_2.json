[["Sep 22 2021 01: +0",0.633,"286"],["Sep 23 2021 01: +0",0.675,"242"],["Sep 24 2021 01: +0",0.65,"189"],["Sep 25 2021 01: +0",0.655,"168"],["Sep 26 2021 01: +0",0.582,"143"],["Sep 27 2021 01: +0",0.572,"118"],["Sep 28 2021 01: +0",0.511,"166"],["Sep 29 2021 01: +0",0.389,"264"],["Sep 30 2021 01: +0",0.468,"232"],["Oct 01 2021 01: +0",0.557,"169"],["Oct 02 2021 01: +0",0.617,"175"],["Oct 03 2021 01: +0",0.658,"166"],["Oct 04 2021 01: +0",0.683,"193"],["Oct 05 2021 01: +0",0.872,"219"],["Oct 06 2021 01: +0",0.717,"318"],["Oct 07 2021 01: +0",0.605,"212"],["Oct 08 2021 01: +0",0.63,"191"],["Oct 09 2021 01: +0",0.636,"182"],["Oct 10 2021 01: +0",0.619,"170"],["Oct 11 2021 01: +0",0.584,"125"],["Oct 12 2021 01: +0",0.597,"205"],["Oct 13 2021 01: +0",0.524,"340"],["Oct 14 2021 01: +0",0.447,"221"],["Oct 15 2021 01: +0",0.41,"198"],["Oct 16 2021 01: +0",0.472,"155"],["Oct 17 2021 01: +0",0.466,"136"],["Oct 18 2021 01: +0",0.466,"115"],["Oct 19 2021 01: +0",0.444,"171"],["Oct 20 2021 01: +0",0.399,"297"],["Oct 21 2021 01: +0",0.416,"200"],["Oct 22 2021 01: +0",0.317,"314"],["Oct 23 2021 01: +0",0.296,"189"],["Oct 24 2021 01: +0",0.294,"120"],["Oct 25 2021 01: +0",0.335,"113"],["Oct 26 2021 01: +0",0.347,"161"],["Oct 27 2021 01: +0",0.266,"238"],["Oct 28 2021 01: +0",0.306,"170"],["Oct 29 2021 01: +0",0.319,"150"],["Oct 30 2021 01: +0",0.314,"158"],["Oct 31 2021 01: +0",0.338,"125"],["Nov 01 2021 01: +0",0.385,"117"],["Nov 02 2021 01: +0",0.423,"146"],["Nov 03 2021 01: +0",0.348,"254"],["Nov 04 2021 01: +0",0.331,"153"],["Nov 05 2021 01: +0",0.347,"134"],["Nov 06 2021 01: +0",0.346,"137"],["Nov 07 2021 01: +0",0.345,"142"],["Nov 08 2021 01: +0",0.333,"123"],["Nov 09 2021 01: +0",0.35,"105"],["Nov 10 2021 01: +0",0.275,"330"],["Nov 11 2021 01: +0",0.276,"192"],["Nov 12 2021 01: +0",0.294,"124"],["Nov 13 2021 01: +0",0.317,"121"],["Nov 14 2021 01: +0",0.323,"118"],["Nov 15 2021 01: +0",0.335,"111"],["Nov 16 2021 01: +0",0.355,"125"],["Nov 17 2021 01: +0",0.339,"188"],["Nov 18 2021 01: +0",0.339,"134"],["Nov 19 2021 01: +0",0.324,"115"],["Nov 20 2021 01: +0",0.331,"125"],["Nov 21 2021 01: +0",0.375,"166"],["Nov 22 2021 01: +0",0.473,"89"],["Nov 23 2021 01: +0",0.421,"89"],["Nov 24 2021 01: +0",0.357,"178"],["Nov 25 2021 01: +0",0.33,"124"],["Nov 26 2021 01: +0",0.308,"158"],["Nov 27 2021 01: +0",0.308,"119"],["Nov 28 2021 01: +0",0.309,"112"],["Nov 29 2021 01: +0",0.309,"110"],["Nov 30 2021 01: +0",0.287,"167"],["Dec 01 2021 01: +0",0.247,"195"],["Dec 02 2021 01: +0",0.262,"147"],["Dec 03 2021 01: +0",0.284,"201"],["Dec 04 2021 01: +0",0.302,"125"],["Dec 05 2021 01: +0",0.309,"89"],["Dec 06 2021 01: +0",0.335,"86"],["Dec 07 2021 01: +0",0.351,"92"],["Dec 08 2021 01: +0",0.283,"189"],["Dec 09 2021 01: +0",0.298,"142"],["Dec 10 2021 01: +0",0.324,"125"],["Dec 11 2021 01: +0",0.315,"99"],["Dec 12 2021 01: +0",0.29,"85"],["Dec 13 2021 01: +0",0.28,"119"],["Dec 14 2021 01: +0",0.304,"76"],["Dec 15 2021 01: +0",0.286,"158"],["Dec 16 2021 01: +0",0.289,"146"],["Dec 17 2021 01: +0",0.292,"99"],["Dec 18 2021 01: +0",0.277,"124"],["Dec 19 2021 01: +0",0.279,"98"],["Dec 20 2021 01: +0",0.278,"83"],["Dec 21 2021 01: +0",0.281,"91"],["Dec 22 2021 01: +0",0.279,"138"],["Dec 23 2021 01: +0",0.263,"127"],["Dec 24 2021 01: +0",0.275,"90"],["Dec 25 2021 01: +0",0.286,"96"],["Dec 26 2021 01: +0",0.297,"101"],["Dec 27 2021 01: +0",0.297,"109"],["Dec 28 2021 01: +0",0.327,"97"],["Dec 29 2021 01: +0",0.335,"168"],["Dec 30 2021 01: +0",0.337,"154"],["Dec 31 2021 01: +0",0.348,"93"],["Jan 01 2022 01: +0",0.351,"113"],["Jan 02 2022 01: +0",0.349,"94"],["Jan 03 2022 01: +0",0.345,"91"],["Jan 04 2022 01: +0",0.337,"121"],["Jan 05 2022 01: +0",0.309,"197"],["Jan 06 2022 01: +0",0.306,"137"],["Jan 07 2022 01: +0",0.321,"145"],["Jan 08 2022 01: +0",0.315,"134"],["Jan 09 2022 01: +0",0.323,"129"],["Jan 10 2022 01: +0",0.333,"120"],["Jan 11 2022 01: +0",0.381,"109"],["Jan 12 2022 01: +0",0.376,"77"],["Jan 13 2022 01: +0",0.398,"87"],["Jan 14 2022 01: +0",0.407,"92"],["Jan 15 2022 01: +0",0.374,"111"],["Jan 16 2022 01: +0",0.36,"91"],["Jan 17 2022 01: +0",0.379,"77"],["Jan 18 2022 01: +0",0.438,"127"],["Jan 19 2022 01: +0",0.43,"71"],["Jan 20 2022 01: +0",0.44,"90"],["Jan 21 2022 01: +0",0.37,"113"],["Jan 22 2022 01: +0",0.375,"130"],["Jan 23 2022 01: +0",0.382,"113"],["Jan 24 2022 01: +0",0.381,"97"],["Jan 25 2022 01: +0",0.424,"99"],["Jan 26 2022 01: +0",0.456,"88"],["Jan 27 2022 01: +0",0.474,"108"],["Jan 28 2022 01: +0",0.487,"125"],["Jan 29 2022 01: +0",0.463,"120"],["Jan 30 2022 01: +0",0.486,"102"],["Jan 31 2022 01: +0",0.5,"113"],["Feb 01 2022 01: +0",0.521,"65"],["Feb 02 2022 01: +0",0.524,"109"],["Feb 03 2022 01: +0",0.452,"87"],["Feb 04 2022 01: +0",0.459,"66"],["Feb 05 2022 01: +0",0.45,"66"],["Feb 06 2022 01: +0",0.491,"95"],["Feb 07 2022 01: +0",0.501,"80"],["Feb 08 2022 01: +0",0.467,"82"],["Feb 09 2022 01: +0",0.475,"72"],["Feb 10 2022 01: +0",0.477,"75"],["Feb 11 2022 01: +0",0.494,"139"],["Feb 12 2022 01: +0",0.685,"211"],["Feb 13 2022 01: +0",0.935,"196"],["Feb 14 2022 01: +0",0.792,"98"],["Feb 15 2022 01: +0",0.718,"74"],["Feb 16 2022 01: +0",0.664,"82"],["Feb 17 2022 01: +0",0.703,"108"],["Feb 18 2022 01: +0",0.697,"97"],["Feb 19 2022 01: +0",0.698,"128"],["Feb 20 2022 01: +0",0.649,"128"],["Feb 21 2022 01: +0",0.599,"82"],["Feb 22 2022 01: +0",0.651,"158"],["Feb 23 2022 01: +0",0.644,"88"],["Feb 24 2022 01: +0",0.589,"66"],["Feb 25 2022 01: +0",0.569,"107"],["Feb 26 2022 01: +0",0.56,"79"],["Feb 27 2022 01: +0",0.52,"71"],["Feb 28 2022 01: +0",0.521,"93"],["Mar 01 2022 01: +0",0.51,"73"],["Mar 02 2022 01: +0",0.541,"92"],["Mar 03 2022 01: +0",0.524,"80"],["Mar 04 2022 01: +0",0.538,"59"],["Mar 05 2022 01: +0",0.601,"47"],["Mar 06 2022 01: +0",0.61,"37"],["Mar 07 2022 01: +0",0.593,"34"],["Mar 08 2022 01: +0",0.609,"43"],["Mar 09 2022 01: +0",0.607,"59"],["Mar 10 2022 01: +0",0.61,"27"],["Mar 11 2022 01: +0",0.602,"40"],["Mar 12 2022 01: +0",0.58,"46"],["Mar 13 2022 01: +0",0.575,"72"],["Mar 14 2022 01: +0",0.618,"64"],["Mar 15 2022 01: +0",0.669,"36"],["Mar 16 2022 01: +0",0.709,"54"],["Mar 17 2022 01: +0",0.713,"30"],["Mar 18 2022 01: +0",0.714,"27"],["Mar 19 2022 01: +0",0.742,"27"],["Mar 20 2022 01: +0",0.716,"32"],["Mar 21 2022 01: +0",0.678,"25"],["Mar 22 2022 01: +0",0.754,"58"],["Mar 23 2022 01: +0",0.772,"29"],["Mar 23 2022 15: +0",0.77,"4"],["Mar 23 2022 16: +0",0.78,"4"],["Mar 23 2022 17: +0",0.782,"2"],["Mar 23 2022 18: +0",0.78,"1"],["Mar 23 2022 19: +0",0.65,"2"],["Mar 23 2022 20: +0",0.849,"38"],["Mar 23 2022 21: +0",0.81,"11"],["Mar 23 2022 22: +0",0.861,"2"],["Mar 24 2022 01: +0",0.755,"2"],["Mar 24 2022 03: +0",0.85,"2"],["Mar 24 2022 05: +0",0.889,"2"],["Mar 24 2022 13: +0",0.84,"1"],["Mar 24 2022 14: +0",0.845,"2"],["Mar 24 2022 16: +0",0.87,"1"],["Mar 24 2022 18: +0",0.91,"5"],["Mar 24 2022 19: +0",0.946,"1"],["Mar 24 2022 20: +0",0.955,"5"],["Mar 24 2022 21: +0",0.862,"1"],["Mar 24 2022 23: +0",0.906,"2"],["Mar 25 2022 01: +0",0.88,"2"],["Mar 25 2022 02: +0",0.73,"1"],["Mar 25 2022 04: +0",0.931,"1"],["Mar 25 2022 06: +0",0.873,"1"],["Mar 25 2022 07: +0",0.87,"1"],["Mar 25 2022 10: +0",0.873,"2"],["Mar 25 2022 11: +0",0.826,"2"],["Mar 25 2022 12: +0",0.609,"1"],["Mar 25 2022 14: +0",0.932,"5"],["Mar 25 2022 15: +0",0.819,"1"],["Mar 25 2022 16: +0",0.87,"1"],["Mar 25 2022 17: +0",0.949,"4"],["Mar 25 2022 20: +0",0.951,"3"],["Mar 25 2022 22: +0",0.93,"2"],["Mar 25 2022 23: +0",0.857,"1"],["Mar 26 2022 03: +0",0.95,"7"],["Mar 26 2022 04: +0",0.97,"1"],["Mar 26 2022 10: +0",0.973,"1"],["Mar 26 2022 12: +0",0.81,"1"],["Mar 26 2022 14: +0",0.948,"2"],["Mar 26 2022 15: +0",0.959,"1"],["Mar 26 2022 16: +0",0.83,"3"],["Mar 26 2022 18: +0",0.944,"1"],["Mar 26 2022 20: +0",0.842,"1"],["Mar 26 2022 21: +0",0.89,"1"],["Mar 26 2022 22: +0",0.84,"7"],["Mar 26 2022 23: +0",0.91,"1"],["Mar 27 2022 01: +0",0.83,"1"],["Mar 27 2022 03: +0",0.85,"1"],["Mar 27 2022 04: +0",0.92,"7"],["Mar 27 2022 11: +0",0.812,"1"],["Mar 27 2022 12: +0",0.88,"1"],["Mar 27 2022 13: +0",0.89,"4"],["Mar 27 2022 14: +0",0.887,"3"],["Mar 27 2022 15: +0",0.89,"2"],["Mar 27 2022 16: +0",0.948,"3"],["Mar 27 2022 17: +0",0.901,"3"],["Mar 27 2022 18: +0",0.896,"2"],["Mar 27 2022 19: +0",0.933,"3"],["Mar 27 2022 20: +0",0.847,"2"],["Mar 27 2022 21: +0",0.958,"4"],["Mar 27 2022 22: +0",0.906,"1"],["Mar 27 2022 23: +0",0.85,"1"],["Mar 28 2022 01: +0",0.956,"1"],["Mar 28 2022 02: +0",0.951,"2"],["Mar 28 2022 06: +0",0.973,"5"],["Mar 28 2022 11: +0",0.962,"5"],["Mar 28 2022 12: +0",0.953,"2"],["Mar 28 2022 14: +0",0.868,"1"],["Mar 28 2022 17: +0",0.878,"5"],["Mar 28 2022 18: +0",0.75,"1"],["Mar 28 2022 19: +0",0.874,"3"],["Mar 28 2022 20: +0",0.878,"5"],["Mar 28 2022 22: +0",0.76,"1"],["Mar 28 2022 23: +0",0.87,"3"],["Mar 29 2022 02: +0",0.919,"1"],["Mar 29 2022 05: +0",0.932,"3"],["Mar 29 2022 10: +0",0.812,"1"],["Mar 29 2022 13: +0",0.823,"6"],["Mar 29 2022 17: +0",0.66,"1"],["Mar 29 2022 18: +0",0.801,"1"],["Mar 29 2022 19: +0",0.78,"9"],["Mar 29 2022 20: +0",0.801,"4"],["Mar 29 2022 21: +0",0.802,"3"],["Mar 29 2022 23: +0",0.796,"1"],["Mar 30 2022 00: +0",0.78,"1"],["Mar 30 2022 04: +0",0.811,"4"],["Mar 30 2022 06: +0",0.82,"1"],["Mar 30 2022 07: +0",0.82,"1"],["Mar 30 2022 08: +0",0.81,"13"],["Mar 30 2022 10: +0",0.781,"1"],["Mar 30 2022 11: +0",0.824,"5"],["Mar 30 2022 12: +0",0.794,"2"],["Mar 30 2022 13: +0",0.871,"1"],["Mar 30 2022 15: +0",0.827,"2"],["Mar 30 2022 16: +0",0.787,"2"],["Mar 30 2022 17: +0",0.868,"5"],["Mar 30 2022 18: +0",0.785,"1"],["Mar 30 2022 19: +0",0.888,"2"],["Mar 30 2022 20: +0",0.78,"1"],["Mar 30 2022 21: +0",0.78,"2"],["Mar 31 2022 00: +0",0.782,"1"],["Mar 31 2022 01: +0",0.891,"1"],["Mar 31 2022 04: +0",0.896,"1"],["Mar 31 2022 06: +0",0.75,"1"],["Mar 31 2022 07: +0",0.897,"1"],["Mar 31 2022 08: +0",0.894,"3"],["Mar 31 2022 09: +0",0.955,"10"],["Mar 31 2022 10: +0",0.896,"2"],["Mar 31 2022 13: +0",0.999,"3"],["Mar 31 2022 14: +0",1.025,"4"],["Mar 31 2022 16: +0",0.96,"1"],["Mar 31 2022 17: +0",0.813,"2"],["Mar 31 2022 18: +0",0.947,"1"],["Mar 31 2022 20: +0",0.891,"2"],["Mar 31 2022 22: +0",0.992,"1"],["Apr 01 2022 07: +0",0.831,"1"],["Apr 01 2022 08: +0",0.821,"1"],["Apr 01 2022 09: +0",0.815,"1"],["Apr 01 2022 13: +0",0.866,"2"],["Apr 01 2022 14: +0",0.94,"6"],["Apr 01 2022 15: +0",0.942,"5"],["Apr 01 2022 16: +0",0.953,"1"],["Apr 01 2022 17: +0",0.886,"1"],["Apr 01 2022 19: +0",0.92,"4"],["Apr 01 2022 20: +0",0.964,"1"],["Apr 01 2022 23: +0",0.85,"1"],["Apr 02 2022 00: +0",0.828,"1"],["Apr 02 2022 02: +0",0.829,"1"],["Apr 02 2022 06: +0",0.884,"2"],["Apr 02 2022 10: +0",0.882,"2"],["Apr 02 2022 11: +0",0.819,"2"],["Apr 02 2022 12: +0",0.872,"4"],["Apr 02 2022 13: +0",0.843,"2"],["Apr 02 2022 14: +0",0.77,"1"],["Apr 02 2022 17: +0",0.903,"2"],["Apr 02 2022 18: +0",0.805,"1"],["Apr 02 2022 19: +0",0.861,"1"],["Apr 02 2022 22: +0",0.851,"5"],["Apr 02 2022 23: +0",0.77,"1"],["Apr 03 2022 01: +0",0.8,"2"],["Apr 03 2022 02: +0",0.89,"1"],["Apr 03 2022 03: +0",0.885,"1"],["Apr 03 2022 04: +0",0.927,"1"],["Apr 03 2022 10: +0",0.945,"1"],["Apr 03 2022 11: +0",0.922,"1"],["Apr 03 2022 12: +0",0.945,"1"],["Apr 03 2022 13: +0",0.881,"1"],["Apr 03 2022 15: +0",0.883,"1"],["Apr 03 2022 18: +0",0.919,"2"],["Apr 03 2022 19: +0",0.804,"1"],["Apr 03 2022 20: +0",0.804,"1"],["Apr 03 2022 21: +0",0.829,"1"],["Apr 03 2022 23: +0",0.779,"2"],["Apr 04 2022 02: +0",0.903,"4"],["Apr 04 2022 03: +0",0.804,"1"],["Apr 04 2022 07: +0",0.897,"1"],["Apr 04 2022 08: +0",0.69,"1"],["Apr 04 2022 09: +0",0.691,"1"],["Apr 04 2022 11: +0",0.912,"2"],["Apr 04 2022 12: +0",0.939,"2"],["Apr 04 2022 15: +0",0.847,"2"],["Apr 04 2022 17: +0",0.861,"2"],["Apr 04 2022 18: +0",0.797,"1"],["Apr 04 2022 19: +0",0.737,"1"],["Apr 04 2022 20: +0",0.85,"3"],["Apr 04 2022 22: +0",0.904,"1"],["Apr 05 2022 03: +0",0.935,"2"],["Apr 05 2022 06: +0",0.794,"1"],["Apr 05 2022 07: +0",0.798,"1"],["Apr 05 2022 08: +0",0.927,"1"],["Apr 05 2022 09: +0",0.928,"1"],["Apr 05 2022 12: +0",0.791,"1"],["Apr 05 2022 13: +0",0.751,"2"],["Apr 05 2022 14: +0",0.833,"1"],["Apr 05 2022 15: +0",0.9,"3"],["Apr 05 2022 17: +0",0.933,"16"],["Apr 05 2022 20: +0",0.93,"4"],["Apr 05 2022 22: +0",0.957,"1"],["Apr 05 2022 23: +0",0.953,"1"],["Apr 06 2022 00: +0",0.947,"2"],["Apr 06 2022 02: +0",0.951,"1"],["Apr 06 2022 07: +0",0.77,"1"],["Apr 06 2022 08: +0",0.88,"2"],["Apr 06 2022 09: +0",0.82,"1"],["Apr 06 2022 13: +0",0.889,"2"],["Apr 06 2022 15: +0",0.865,"1"],["Apr 06 2022 18: +0",0.909,"11"],["Apr 06 2022 19: +0",0.931,"1"],["Apr 06 2022 20: +0",0.95,"3"],["Apr 06 2022 22: +0",0.81,"1"],["Apr 06 2022 23: +0",0.78,"1"],["Apr 07 2022 02: +0",0.75,"1"],["Apr 07 2022 10: +0",0.76,"1"],["Apr 07 2022 12: +0",0.754,"1"],["Apr 07 2022 13: +0",0.909,"4"],["Apr 07 2022 14: +0",0.931,"5"],["Apr 07 2022 16: +0",0.893,"1"],["Apr 07 2022 18: +0",0.91,"2"],["Apr 07 2022 19: +0",0.922,"1"],["Apr 07 2022 20: +0",0.628,"1"],["Apr 07 2022 21: +0",0.945,"1"],["Apr 07 2022 22: +0",0.892,"1"],["Apr 08 2022 01: +0",0.921,"1"],["Apr 08 2022 03: +0",0.916,"1"],["Apr 08 2022 04: +0",0.921,"2"],["Apr 08 2022 07: +0",0.912,"1"],["Apr 08 2022 08: +0",0.82,"1"],["Apr 08 2022 09: +0",0.819,"2"],["Apr 08 2022 11: +0",0.944,"1"],["Apr 08 2022 13: +0",0.817,"1"],["Apr 08 2022 14: +0",0.883,"3"],["Apr 08 2022 15: +0",0.93,"4"],["Apr 08 2022 16: +0",0.953,"1"],["Apr 08 2022 17: +0",0.956,"2"],["Apr 08 2022 18: +0",0.96,"1"],["Apr 08 2022 19: +0",0.858,"4"],["Apr 08 2022 20: +0",0.911,"1"],["Apr 08 2022 21: +0",0.91,"1"],["Apr 08 2022 22: +0",0.949,"2"],["Apr 09 2022 00: +0",0.967,"1"],["Apr 09 2022 01: +0",0.97,"3"],["Apr 09 2022 03: +0",0.97,"6"],["Apr 09 2022 06: +0",0.941,"1"],["Apr 09 2022 08: +0",0.786,"1"],["Apr 09 2022 09: +0",0.976,"5"],["Apr 09 2022 10: +0",0.988,"1"],["Apr 09 2022 12: +0",0.982,"1"],["Apr 09 2022 14: +0",0.961,"1"],["Apr 09 2022 17: +0",0.894,"4"],["Apr 09 2022 18: +0",0.996,"1"],["Apr 09 2022 19: +0",1.008,"1"],["Apr 09 2022 20: +0",0.867,"2"],["Apr 09 2022 21: +0",0.823,"3"],["Apr 09 2022 22: +0",0.973,"1"],["Apr 09 2022 23: +0",0.896,"2"],["Apr 10 2022 00: +0",0.964,"3"],["Apr 10 2022 01: +0",0.99,"4"],["Apr 10 2022 04: +0",0.75,"1"],["Apr 10 2022 08: +0",0.867,"1"],["Apr 10 2022 10: +0",0.854,"1"],["Apr 10 2022 11: +0",0.823,"1"],["Apr 10 2022 13: +0",0.888,"1"],["Apr 10 2022 14: +0",0.892,"2"],["Apr 10 2022 15: +0",0.888,"5"],["Apr 10 2022 19: +0",0.78,"2"],["Apr 11 2022 00: +0",0.78,"1"],["Apr 11 2022 08: +0",0.78,"1"],["Apr 11 2022 09: +0",0.638,"1"],["Apr 11 2022 10: +0",0.84,"1"],["Apr 11 2022 12: +0",0.861,"1"],["Apr 11 2022 13: +0",0.82,"1"],["Apr 11 2022 14: +0",0.833,"1"],["Apr 11 2022 15: +0",0.84,"1"],["Apr 11 2022 18: +0",0.747,"2"],["Apr 11 2022 19: +0",0.82,"3"],["Apr 11 2022 21: +0",0.849,"2"],["Apr 11 2022 22: +0",0.8,"1"],["Apr 11 2022 23: +0",0.892,"1"],["Apr 12 2022 00: +0",0.8,"1"],["Apr 12 2022 01: +0",0.855,"1"],["Apr 12 2022 04: +0",0.908,"1"],["Apr 12 2022 06: +0",0.913,"1"],["Apr 12 2022 08: +0",0.913,"1"],["Apr 12 2022 11: +0",0.867,"1"],["Apr 12 2022 12: +0",0.87,"1"],["Apr 12 2022 14: +0",0.87,"1"],["Apr 12 2022 17: +0",0.875,"1"],["Apr 12 2022 20: +0",0.794,"2"],["Apr 12 2022 21: +0",0.848,"1"],["Apr 12 2022 23: +0",0.882,"2"],["Apr 13 2022 01: +0",0.881,"1"],["Apr 13 2022 07: +0",0.889,"1"],["Apr 13 2022 09: +0",0.85,"2"],["Apr 13 2022 10: +0",0.794,"1"],["Apr 13 2022 13: +0",0.896,"1"],["Apr 13 2022 14: +0",1.003,"41"],["Apr 13 2022 16: +0",0.89,"1"],["Apr 13 2022 17: +0",0.987,"8"],["Apr 13 2022 18: +0",0.807,"1"],["Apr 13 2022 21: +0",0.918,"6"],["Apr 14 2022 03: +0",0.81,"1"],["Apr 14 2022 05: +0",0.96,"1"],["Apr 14 2022 08: +0",0.998,"3"],["Apr 14 2022 10: +0",1.037,"2"],["Apr 14 2022 11: +0",0.841,"1"],["Apr 14 2022 12: +0",1.027,"1"],["Apr 14 2022 13: +0",1.058,"1"],["Apr 14 2022 14: +0",0.992,"1"],["Apr 14 2022 16: +0",1.005,"1"],["Apr 14 2022 18: +0",0.857,"2"],["Apr 14 2022 19: +0",0.922,"2"],["Apr 14 2022 20: +0",0.843,"1"],["Apr 14 2022 23: +0",0.99,"2"],["Apr 15 2022 00: +0",0.97,"1"],["Apr 15 2022 01: +0",0.994,"1"],["Apr 15 2022 05: +0",0.96,"1"],["Apr 15 2022 08: +0",1.012,"2"],["Apr 15 2022 09: +0",1.017,"1"],["Apr 15 2022 10: +0",1.022,"1"],["Apr 15 2022 13: +0",1.028,"1"],["Apr 15 2022 16: +0",0.861,"1"],["Apr 15 2022 17: +0",0.929,"2"],["Apr 15 2022 18: +0",0.692,"1"],["Apr 15 2022 19: +0",1.006,"1"],["Apr 15 2022 20: +0",0.99,"1"],["Apr 15 2022 21: +0",1.014,"2"],["Apr 15 2022 22: +0",0.83,"3"],["Apr 16 2022 00: +0",0.85,"1"],["Apr 16 2022 03: +0",0.824,"2"],["Apr 16 2022 06: +0",0.897,"3"],["Apr 16 2022 08: +0",0.975,"4"],["Apr 16 2022 09: +0",1.06,"1"],["Apr 16 2022 12: +0",1.062,"2"],["Apr 16 2022 13: +0",1.073,"4"],["Apr 16 2022 14: +0",1.076,"3"],["Apr 16 2022 17: +0",1.062,"1"],["Apr 16 2022 18: +0",1.06,"1"],["Apr 16 2022 22: +0",1.079,"10"],["Apr 16 2022 23: +0",1.019,"2"],["Apr 17 2022 07: +0",1.098,"1"],["Apr 17 2022 13: +0",0.936,"2"],["Apr 17 2022 14: +0",0.94,"1"],["Apr 17 2022 15: +0",0.88,"1"],["Apr 17 2022 16: +0",0.897,"1"],["Apr 17 2022 18: +0",1.063,"5"],["Apr 17 2022 19: +0",1.136,"4"],["Apr 17 2022 21: +0",1.006,"1"],["Apr 17 2022 22: +0",1.017,"1"],["Apr 17 2022 23: +0",1.16,"1"],["Apr 18 2022 07: +0",1.006,"1"],["Apr 18 2022 10: +0",0.89,"1"],["Apr 18 2022 11: +0",1.015,"1"],["Apr 18 2022 13: +0",1.015,"4"],["Apr 18 2022 16: +0",1.01,"4"],["Apr 18 2022 17: +0",0.89,"1"],["Apr 18 2022 18: +0",0.961,"1"],["Apr 18 2022 20: +0",0.94,"1"],["Apr 18 2022 22: +0",0.923,"2"],["Apr 19 2022 00: +0",0.87,"1"],["Apr 19 2022 01: +0",0.875,"3"],["Apr 19 2022 02: +0",0.99,"1"],["Apr 19 2022 10: +0",0.986,"1"],["Apr 19 2022 11: +0",0.989,"3"],["Apr 19 2022 12: +0",0.982,"1"],["Apr 19 2022 14: +0",0.864,"1"],["Apr 19 2022 15: +0",0.962,"1"],["Apr 19 2022 16: +0",0.89,"1"],["Apr 19 2022 18: +0",0.94,"5"],["Apr 19 2022 19: +0",1.005,"10"],["Apr 19 2022 20: +0",1.024,"7"],["Apr 19 2022 21: +0",1.185,"29"],["Apr 20 2022 00: +0",1.186,"2"],["Apr 20 2022 02: +0",0.89,"1"],["Apr 20 2022 08: +0",1.302,"2"],["Apr 20 2022 09: +0",0.89,"1"],["Apr 20 2022 11: +0",1.134,"1"],["Apr 20 2022 12: +0",1.08,"3"],["Apr 20 2022 14: +0",1.286,"3"],["Apr 20 2022 18: +0",1.289,"2"],["Apr 20 2022 19: +0",1.29,"2"],["Apr 20 2022 20: +0",0.88,"1"],["Apr 21 2022 04: +0",0.991,"1"],["Apr 21 2022 10: +0",0.995,"1"],["Apr 21 2022 11: +0",1.122,"2"],["Apr 21 2022 17: +0",1.133,"4"],["Apr 21 2022 19: +0",1,"1"],["Apr 22 2022 05: +0",1.08,"1"],["Apr 22 2022 06: +0",0.975,"1"],["Apr 22 2022 07: +0",0.84,"1"],["Apr 22 2022 12: +0",1.139,"5"],["Apr 22 2022 15: +0",1.257,"4"],["Apr 22 2022 18: +0",1.152,"1"],["Apr 22 2022 20: +0",1.001,"1"],["Apr 22 2022 21: +0",1.05,"1"],["Apr 22 2022 22: +0",1.092,"2"],["Apr 23 2022 01: +0",1.01,"2"],["Apr 23 2022 03: +0",0.989,"1"],["Apr 23 2022 04: +0",0.97,"1"],["Apr 23 2022 05: +0",1.032,"3"],["Apr 23 2022 07: +0",0.97,"1"],["Apr 23 2022 08: +0",1.211,"1"],["Apr 23 2022 10: +0",1.254,"5"],["Apr 23 2022 11: +0",1.276,"7"],["Apr 23 2022 13: +0",1.211,"1"]]