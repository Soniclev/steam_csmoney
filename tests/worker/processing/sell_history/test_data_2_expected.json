{
  "pairs": {
    "0.82": 100.0,
    "0.83": 100.0,
    "0.84": 98.0,
    "0.85": 98.0,
    "0.86": 97.0,
    "0.87": 97.0,
    "0.88": 96.0,
    "0.89": 94.0,
    "0.9": 91.0,
    "0.92": 89.0,
    "0.94": 88.0,
    "0.96": 84.0,
    "0.97": 83.0,
    "0.98": 80.0,
    "0.99": 79.0,
    "1.0": 75.0,
    "1.01": 69.0,
    "1.02": 61.0,
    "1.03": 56.0,
    "1.05": 55.0,
    "1.06": 54.0,
    "1.07": 49.0,
    "1.08": 47.0,
    "1.09": 39.0,
    "1.1": 38.0,
    "1.12": 38.0,
    "1.13": 37.0,
    "1.14": 34.0,
    "1.15": 30.0,
    "1.16": 29.0,
    "1.19": 29.0,
    "1.21": 14.0,
    "1.25": 13.0,
    "1.26": 10.0,
    "1.28": 8.0,
    "1.29": 5.0,
    "1.3": 1.0
  }
}